.github/workflows/reusable-ecamp3-deployment.yml

```
@@ -0,0 +1,180 @@
+      dropDBOnUninstall:
+        required: false
+        type: boolean
+        default: false
+      restoreSourceFile:
+        required: false
+        type: string
+      action:
+        description: "Choose action of (diff|deploy)"
+        type: string
+        required: true
+        default: diff
```

actionlint (1.7.7)

input "action" of workflow_call event has the default value "diff", but it is also required. if an input is marked as required, its default value will never be used

(events)